Travis build is broken

This is related to a change in OpenLayers.

```
node buildtools/build.js buildtools/ngeo.json dist/ngeo.js

info ol Parsing dependencies

info ol Compiling 406 sources

ERR! compile /home/travis/build/camptocamp/ngeo/src/ol-ext/format/featurehash.js:919: ERROR - actual parameter 1 of ngeo.format.FeatureHash.encodeStyles_ does not match formal parameter

ERR! compile found   : (Array<(null|ol.style.Style)>|ol.style.Style)

ERR! compile required: (Array<(null|ol.style.Style)>|null)

ERR! compile             styles, geometry.getType(), encodedStyles);

ERR! compile             ^

ERR! compile 

ERR! compile /home/travis/build/camptocamp/ngeo/src/services/featureoverlay.js:184: ERROR - inconsistent return type

ERR! compile found   : (Array<(null|ol.style.Style)>|null|ol.style.Style)

ERR! compile required: (Array<(null|ol.style.Style)>|null)

ERR! compile   return group.styleFunction(feature, resolution);

ERR! compile          ^

ERR! compile 

ERR! compile 

ERR! compile 2 error(s), 0 warning(s), 

ERR! compile 97.4% typed

ERR! compile 

ERR! Process exited with non-zero status, see log for more detail: 2 

make: *** [dist/ngeo.js] Error 1
```

--- a/src/ol-ext/format/featurehash.js
+++ b/src/ol-ext/format/featurehash.js
@@ -915,6 +915,7 @@ ngeo.format.FeatureHash.prototype.writeFeatureText =
       var styles = styleFunction.call(feature, 0);
       if (!goog.isNull(styles)) {
         var encodedStyles = [];
+        styles = goog.isArray(styles) ? styles : [styles];
         ngeo.format.FeatureHash.encodeStyles_(
             styles, geometry.getType(), encodedStyles);
         if (encodedStyles.length > 0) {

--- a/src/services/featureoverlay.js
+++ b/src/services/featureoverlay.js
@@ -172,7 +172,7 @@ ngeo.FeatureOverlayMgr.prototype.setStyle = function(style, groupIndex) {
 /**
  * @param {ol.Feature|ol.render.Feature} feature Feature.
  * @param {number} resolution Resolution.
- * @return {Array.<ol.style.Style>} Styles.
+ * @return {Array.<ol.style.Style>|ol.style.Style} Styles.
  * @private
  */
 ngeo.FeatureOverlayMgr.prototype.styleFunction_ =

